patroni/ctl.py

```diff
@@ -1088,9 +1089,18 @@ def invoke_editor(before_editing, cluster_name):
     :param before_editing: human representation before editing
     :returns tuple of human readable and parsed datastructure after changes
     """
-    editor_cmd = os.environ.get('EDITOR')
+    if 'EDITOR' in os.environ:
+        editor_cmd = os.environ.get('EDITOR')
+    elif os.path.exists('/usr/bin/editor'):
```

Comment:
I think it is better to avoid assumptions about the location of binary files. Why not use `find_executable('editor')` here?
We can also make a loop here:
```python
for editor in ('editor', 'vi'):
    editor_cmd = find_executable(editor)
    if editor_cmd:
        logging.debug('Setting fallback editor_cmd=%s', editor_cmd)
        break
```